builtin/providers/aws/import_aws_iam_role_test.go

```diff
@@ -0,0 +1,28 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSIAMRole_importBasic(t *testing.T) {
```

Comment:
This should be `TestAccAWSRole_importBasic` - then it will get run with the other IAM Role tests :)